### PR TITLE
Update ReadMe to include alert in Packages Requiring I18n list

### PR DESCRIPTION
**Pull request by dave-roth**

### CODE REVIEW

**Comment by benbcai** [comment]
It might be good to add a Component Features section and put Localization/Internationalization Support  under it just like [Demog Banner](https://github.com/cerner/terra-core/blob/master/packages/terra-demographics-banner/docs/README.md#component-features) and [DatePicker](https://github.com/cerner/terra-core/tree/Issue-246-internationalize-date-picker/packages/terra-date-picker/docs#component-features).

**Comment by dave-roth** [comment]
Done


CONFIDENTIALITY NOTICE This message and any included attachments are from Cerner Corporation and are intended only for the addressee. The information contained in this message is confidential and may constitute inside or non-public information under international, federal, or state securities laws. Unauthorized forwarding, printing, copying, distribution, or use of such information is strictly prohibited and may be unlawful. If you are not the addressee, please promptly delete this message and notify the sender of the delivery error by e-mail or you may call Cerner's corporate offices in Kansas City, Missouri, U.S.A at (+1) (816)221-1024.

File: packages/terra-alert/docs/README.md [hunk 1]
```diff
@@ -39,3 +39,15 @@ import Alert from 'terra-alert';
...
+The Alert component must be composed inside the [Base][1] component with a locale in order for it to load the correct translation strings. These translation strings are used for the default alert title when the application does not specify a title for the alert.
+
+[1]: https://github.com/cerner/terra-core/tree/master/packages/terra-base/docs
+
+## Component Features
+* [Cross-Browser Support](https://github.com/cerner/terra-core/wiki/Component-Features#cross-browser-support)
+* [Responsive Support](https://github.com/cerner/terra-core/wiki/Component-Features#responsive-support)
+* [Mobile Support](https://github.com/cerner/terra-core/wiki/Component-Features#mobile-support)
+* [Internationalization Support](https://github.com/cerner/terra-core/wiki/Component-Features#internationalization-i18n-support)
+* [Localization Support](https://github.com/cerner/terra-core/wiki/Component-Features#localization-support)
```
Comment (StephenEsser) [inline packages/terra-alert/docs/README.md]: Do updates to the ReadMe constitute enough of a change to update the change log?

Comment (dave-roth) [inline packages/terra-alert/docs/README.md]: Updated Change Log

**Pull request merged**